sdk/keyvault/azure-security-test-keyvault-jca/src/test/java/com/azure/security/keyvault/jca/KeyVaultKeyManagerTest.java

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.security.keyvault.jca;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledIfEnvironmentVariable;
+
+import java.io.IOException;
+import java.security.KeyStore;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.Security;
+import java.security.cert.CertificateException;
+import java.util.Arrays;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+@EnabledIfEnvironmentVariable(named = "AZURE_KEYVAULT_CERTIFICATE_NAME", matches = "myalias")
+public class KeyVaultKeyManagerTest {
+
+    private static KeyVaultKeyManager manager;
+    private static String certificateName;
+
+    @BeforeAll
+    public static void setEnvironmentProperty() throws KeyStoreException, NoSuchAlgorithmException, IOException,
+        CertificateException {
+        PropertyConvertorUtils.putEnvironmentPropertyToSystemProperty(
```

Comment:
Need this action to be performed each time a test is scheduled?

Reply:
It's execute it only once before running all tests in one class, it‘s better than @BeforeEach in our case.